HotCommands/CommandFilter.cs

```diff
@@ -60,6 +64,8 @@ public int QueryStatus(ref Guid pguidCmdGroup, uint cCmds, OLECMD[] prgCmds, Int
                 {
                     case Constants.ToggleCommentCmdId:
                     case Constants.ExpandSelectionCmdId:
+                    case Constants.MoveMemberUpCmdId:
+                    case Constants.MoveMemberDownCmdId:
```

Comment:
That'll handle the package, but not the commands. Essentially, that would just mean that the package would load once a CSharpProject is loaded. However, once the package is loaded, it would still operate on other files. There are many ways to address this. The QueryStatus for the commands is a pretty simple place to check the content type of the underlying buffer. We use code like [this](https://github.com/dotnet/roslyn/blob/56f605c41915317ccdb925f66974ee52282609e7/src/EditorFeatures/Core/Shared/Extensions/ITextViewExtensions.cs#L69-L73) to get the subject buffer containing the caret. IIRC, the commands can also be registered with C# as a UI Context, but I don't know much about that.